src/functional_test_runner/lib/proc_runner/log.js

```diff
@@ -0,0 +1,4 @@
+import { createToolingLog } from '@kbn/plugin-helpers';
```

Comment:
`createToolingLog` is calling through to code that already lives in the Kibana repo (see https://github.com/elastic/kibana/blob/5ecdaa7e7b13dce0c824ded1a26abbdfc5d89295/packages/kbn-plugin-helpers/lib/utils.js#L25-L27). But maybe we want to extract it, as I mentioned above ^.

Reply:
cc @spalger 

Let's talk about `@kbn/tooling-log` possibilities. Looking at the code, seems modularized enough to pull out easily into a package. It's used moderately around kibana code, so it merits its own PR to track those differences.

Meanwhile, can we get this PR up merged up just by referencing the plugin-helpers module via `"@kbn/plugin-helpers": "link:../kbn-plugin-helpers",` in `package.json` for `@kbn/proc-runner`? Internally, `@kbn/plugin-helpers`'s `tooling_log` actually comes from kibana src/dev, so that should be fine for now. Follow up PR can be right after.

Reply:
Didn't see this, I'd really like tooling log and proc runner to live together, more discussion below: https://github.com/elastic/kibana/pull/17120#discussion_r174192815